src/Raven.Server/Documents/Replication/Incoming/AbstractIncomingReplicationHandler.cs

```diff
@@ -199,10 +214,9 @@ protected void ReceiveReplicationBatches()
                                 }
                                 else
                                 {
-                                    //if we are disposing, do not notify about failure (not relevant)
+                                    // if we are disposing, do not notify about failure (not relevant)
                                     if (_cts.IsCancellationRequested == false)
-                                        if (Logger.IsInfoEnabled)
-                                            Logger.Info("Received unexpected exception while receiving replication batch.", e);
+                                        Logger.Info("Received unexpected exception while receiving replication batch.", e);
```

Comment:
if (Logger.IsInfoEnabled)

Reply:
We are already inside that `if` statement.

Reply:
you can also remove it from `HandleSocketException`